includes/class-freemius.php

```diff
@@ -23248,7 +23279,24 @@ function _affiliation_page_render() {
 
             fs_enqueue_local_style( 'fs_affiliation', '/admin/affiliation.css' );
 
-            $vars = array( 'id' => $this->_module_id );
+            $is_bundle_context = ! is_null( $this->get_bundle_id() );
+
+            $plugin_title = $this->get_plugin_title();
+            if ( $is_bundle_context ) {
+                $plugin_title = $this->plugin_affiliate_terms->plugin_title;
+                // Add the suffix "Bundle" only if the word is not present in the title itself.
+                if ( false === mb_stripos( $plugin_title, fs_text_inline( 'Bundle', 'bundle' ) ) ) {
+                    $plugin_title = $this->apply_filters(
+                        'formatted_bundle_title',
+                        $plugin_title . fs_text_inline( ' Bundle', 'bundle-suffix' )

```

Comment:
@swashata I'd propose to use `fs_text_inline( 'Bundle', 'bundle' )` and add the space outside the transaction function. I understand how this provides more flexibility, but IMO it's too much. If someone will asks us for it, we can always enrich it later.